i18n/fa.yaml

```diff
@@ -1,19 +1,24 @@
 - id: prev_page
-  translation: "صفحه قبلی"
+  translation: "صفحه قبل"
 
 - id: next_page
-  translation: "صفحه بعدی"
+  translation: "صفحه بعد"
 
 - id: read_time
   translation:
-    one: "۱ دقیقه"
+    one : "1 دقیقه"
```

Comment:
Why would you want to change this? I think it's better to keep using the Persian numbers.

Reply:
@hialisabet Okay, I didn't notice it. But then I think it'd make sense to change the plural forms to show the Persian numbers.